### PR TITLE
doc: add vmemcache man page to pmem.io

### PR DESCRIPTION
**Pull request by ldorau**
This change is [Reviewable](https://reviewable.io/reviews/pmem/vmemcache/218)

### CODE REVIEW

**Comment by ldorau** [comment]
@wlemkows please review

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+libvmemcache
+============
+
+### pmem.io/vmemcache web site
+
+This repo contains documentation for libvmemcache.
+
+To view documentation go to: [manpages](http://pmem.io/vmemcache/vmemcache.3.md)
```
Comment (wlemkows) [inline README.md]: This path is wrong (on pmdk-convert it is wrong too)

Comment (ldorau) [reply README.md]: Done

**Comment by ldorau** [comment]
@lukaszstolarczuk please resolve your issues

**Comment by ldorau** [comment]
@wlemkows please review 2 files left

**Pull request merged**